src/ngx_http_lua_util.h

```diff
@@ -169,6 +172,9 @@ void ngx_http_lua_unescape_uri(u_char **dst, u_char **src, size_t size,
 uintptr_t ngx_http_lua_escape_uri(u_char *dst, u_char *src,
     size_t size, ngx_uint_t type);
 
+ngx_int_t ngx_http_lua_copy_escaped_header(ngx_http_request_t *r,
+    ngx_str_t *dst, ngx_uint_t header_name);
```

Comment:
We usually use `int` or `unsigned` for boolean types (the only exception is in the nginx directive configuration where we use `ngx_flag_t` for boolean config values). So `int` here is better for `is_name` (BTW, it should be `is_name` than `header_name`).

`ngx_uint_t` is like `unsigned long` on x64, BTW, which is 8 bytes long.